src/components/FolderCreationModal.jsx

```diff
@@ -26,11 +27,10 @@ const FolderCreationModal = ({
   pagesData,
   siteName,
   setIsFolderCreationActive,
-  setRedirectToNewPage,
```

Comment:
update proptypes for this component to remove `setRedirectToNewPage`, `setNewPageUrl`

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/359/commits/522b998ef4cf177516b136488f58a9abf882989c